src/common/helpers/id.ts

```diff
@@ -0,0 +1,13 @@
+export const random = (length = 8) => {
```

Comment:
We should never generate any IDs in the React app. IDs are optional for the first request and they'll be filled upon next one.